docs/content/preview/contribute/docs/macos.md

```diff
@@ -0,0 +1,25 @@
+<!--
++++
+private=true
++++
+-->
+
+* **Command-line tools for Xcode** on macOS.
+
+    ```sh
+    $ xcode-select --install
+    ```
+
+    Xcode is many gigabytes. Install the command-line tools unless you actually need the full Xcode.
+
+* [**Homebrew**](https://brew.sh) on macOS.
+
+* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.
+
+    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python version 3.10 or earlier.
+
+* **Hugo**: `brew install hugo` installs the latest version.
```

Comment:
So for MacOS, add these instructions to install a specific version. Lets fix 0.125.6 as thats the current one we are using :
Include the following steps to install a specific hugo version. (0.125.6)
• Unpin hugo - brew unpin hugo
• Uninstall older version - brew uninstall hugo
• Download the attached file.
• brew install hugo.rb
• Prevent automatic updates with brew pin hugo

Reply:
Added the steps for v0.131.0 version of Hugo. Please check.